src/NovaSDS011.h

```diff
@@ -41,7 +41,7 @@ enum QuerryError
 
 enum WorkingMode
 {
-	sleep = 0,
+	_sleep = 0,
```

Comment:
Pleas stick to one naming style for whole enum. 
Lets go with mode_sleep, mode_work, mode_work_error.